### PR TITLE
Module-1-Task-1 OOP

### PR DESCRIPTION
**Pull request by Shibbaz**

### CODE REVIEW

File: config/master.key [hunk 1]
```diff
@@ -0,0 +1 @@
+3f06de68337649378fd19b4403f77c24
```
Comment (DenisNikolski) [inline config/master.key:1]: this is secret key, that should not been pushed to github 
you can read more about it here
https://medium.com/@thorntonbrenden/rails-and-the-legendary-master-key-15c8be7799f1

so `config/master.key` shoud be added to `.gitignore` file, 
here good example Rails specific `.gitignore` file
https://github.com/github/gitignore/blob/main/Rails.gitignore

in real production projects, you should not push any secrets to repository,
if they was accidentally pushed --> they should be rotated

File: log/development.log [hunk 2]
```diff
@@ -0,0 +1,43 @@
+  [1m[35m (112.6ms)[0m  [1m[35mCREATE DATABASE "films_imdb_development" ENCODING = 'unicode'[0m
```
Comment (DenisNikolski) [inline log/development.log:1]: as well, we don't need to commit log `folder`
please update `.gitignore` file 

this folder contains local development logs, so there is no value in it

File: tmp/development_secret.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+8ce0828687d832e7cecc3de17fea645cf628548bc9aa557782316eb73de9b3d8733007ea23a9cbc0f4cafaec17402f44b8a07af27048a9a5889cca9e873a4fa7
```
Comment (DenisNikolski) [inline tmp/development_secret.txt:1]: as well, this should not be commited

**Review by DenisNikolski — COMMENTED** [review]
please update `.gitingnore ` and delete unnecessary files and folders

File: config/master.key (hunk 1 above)
Comment (Shibbaz) [reply config/master.key:1]: True, something happened with the gitignore, gonna update it.

**Review by DenisNikolski — APPROVED** [review]

**Pull request merged**